AU2/plugins/CorePlugin.py

```diff
@@ -226,13 +234,12 @@ def on_assassin_update(self, assassin: Assassin, htmlResponse: Dict) -> List[HTM
         [assassin.add_pseudonym(u.text, u.valid_from) for u in pseudonym_updates.new_values]
         [assassin.edit_pseudonym(i, v.text, v.valid_from) for i, v in pseudonym_updates.edited.items()]
         [assassin.delete_pseudonym(i) for i in pseudonym_updates.deleted_indices]
-
+        # set other attributes
         assassin.real_name = htmlResponse[self.html_ids["Real Name"]]
         assassin.pronouns = htmlResponse[self.html_ids["Pronouns"]]
         assassin.address = htmlResponse[self.html_ids["Address"]]
         assassin.college = htmlResponse[self.html_ids["College"]]
         assassin.notes = htmlResponse[self.html_ids["Notes"]]
-        #assassin.is_police = htmlResponse[self.html_ids["Police"]]

```

Comment:
(same as above) Maybe leave another comment explaining why this is commented, or just remove the comment altogether. Not obvious why it's there otherwise